ide/parsing.indexing/src/org/netbeans/modules/parsing/impl/indexing/PathRegistry.java

```diff
@@ -68,7 +68,7 @@
 public final class PathRegistry implements Runnable {
 
     private static final boolean FIRE_UNKNOWN_ALWAYS = false;
-    /*test*/ static final int FIRER_EVT_COLLAPSE_WINDOW = 500;
+    /*test*/ static int FIRER_EVT_COLLAPSE_WINDOW = 500;
```

Comment:
Suggestion: Set this from a system property (Full classname + Fieldname?) and default to 500. That way the field can be kept static final and there is no need to twiddle with accessibility of the field.

Reply:
sure. This was only rudimentary. Will change it.

Reply:
done